libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm/primarySchoolSection/newSchoolSubSection.ts

```diff
@@ -15,6 +16,11 @@ import { FriggSchoolsByMunicipalityQuery } from '../../../types/schema'
 export const newSchoolSubSection = buildSubSection({
   id: 'newSchoolSubSection',
   title: newPrimarySchoolMessages.primarySchool.newSchoolSubSectionTitle,
+  condition: (answers) => {
+    // Only display section if not seleting neighbourhood school
+    const { applyForNeighbourhoodSchool } = getApplicationAnswers(answers)
+    return applyForNeighbourhoodSchool === NO
+  },
```

Comment:
This page should also appear if `NEW_PRIMARY_SCHOOL`.
```suggestion
  condition: (answers) => {
    // Only display section if not seleting neighbourhood school
    const { applicationType, applyForNeighbourhoodSchool } =
      getApplicationAnswers(answers)
    return (
      applicationType === ApplicationType.NEW_PRIMARY_SCHOOL ||
      (applicationType === ApplicationType.ENROLLMENT_IN_PRIMARY_SCHOOL &&
        applyForNeighbourhoodSchool === NO)
    )
  },
```

Reply:
Fixed